src/exportImage.py

```diff
@@ -121,49 +147,92 @@ def delete_local_directory(self, directoryName):
         self.log.info(
             f'Folder "{directoryName}" in path: "{directory_path}" removed successfully')
 
-    def create_geopackage(self, bbox, filename, url, taskid, fullPath, resolution):
+    def create_geopackage(self, bbox, filename, url, task_id, full_path, max_zoom, resolution):
+        warp_result = self.create_geopackage_base(
+            bbox, filename, url, task_id, full_path, resolution)
+
+        if not warp_result:
+            self.log.error(f'gdal return empty response for task: "{task_id}"')
+            return False
+
+        overviews_result = self.create_geopackage_overview(
+            filename, task_id, full_path, max_zoom)
+
+        # Clear cache (for closing process)
```

Comment:
should'nt it be before the "create_geopackage_overview" method?

Reply:
You are correct